docs/content/docs/guide/statuses.md

```diff
@@ -32,8 +32,6 @@ Example:
 /test <pipelineRun-name>
 ```
 
-NOTE: `/test` is currently supported only on GitHub Apps.
-
```

Comment:
is /test all supported ?

Reply:
not yet
https://github.com/openshift-pipelines/pipelines-as-code/issues/662
have a issue for that